moon/rospy/rospy_base.py

```diff
@@ -19,6 +19,7 @@
 
 import rospy
 from rosgraph_msgs.msg import Clock
+from std_msgs.msg import *
```

Comment:
in general import * is not recommended, i.e. I would list the messages you really use

Reply:
ok, changed